features/test-project.feature

```diff
@@ -0,0 +1,152 @@
+Feature: Test Mocha Project
+  
+  Scenario: Testing a project
+    Given I visit sample project file "sample-project/test/parse-float-test.js"
+    When I switch to buffer "parse-float-test.js"
+    Then I should be in buffer "parse-float-test.js"
+    When I run the command "mocha-test-project"
+    And I wait for the compilation to finish
+    Then I should see buffer "*mocha tests*"
+    And I should see contents in buffer "*mocha tests*":
+      """
+      Mocha started
+      """
+      """
+      2 passing
+      """
+      """
+      1 failing
+      """
+      """
+      Mocha exited abnormally
+      """
```

Comment:
I think they way you changed to is best, ie:

Then I should see ...
And I should see ...
And I should see ...
....